apps/charterafrica/src/lib/data/index.js

```diff
@@ -102,6 +129,60 @@ async function processPageResearch({ blocks }) {
   });
 }
 
+async function processPageCommunity({ blocks }) {
```

Comment:
Can you point me to this community page in Figma? I only see News and Research under Knowledge. The community page under Opportunity looks like 👇🏽 . No articles or feature posts.

![Screenshot from 2023-02-09 15-50-03](https://user-images.githubusercontent.com/1779590/217817602-507c64db-e35f-4986-9fbd-092c98d0d4a3.png)